raiden/network/transport/matrix/transport.py

```diff
@@ -1159,27 +1184,27 @@ def _set_room_id_for_address(self, address: Address, room_id: Optional[_RoomID]
         If room_id is falsy, clean list of rooms. Else, push room_id to front of the list """
 
         assert not room_id or room_id in self._client.rooms, "Invalid room_id"
-        address_hex: AddressHex = to_checksum_address(address)
+        address_checksum: ChecksumAddress = to_checksum_address(address)
```

Comment:
I think we should stick with `AddressHex` here, it's the type used everywhere in the client, even though it's nto the return type of `to_checksum_address`. But in the future we can make `AddressHex` an alias for `ChecksumAddress`.